src/main/scala/de/htwg/se/NineMensMorris/model/gameboardComponent/GameboardInterface.scala

```diff
@@ -26,6 +26,7 @@ trait EdgeInterface {
 trait FieldInterface {
   def id: Int
   def fieldStatus: FieldStatus
+  def neighList: List[(_, _)]
```

Comment:
die "neighList" sollte hier die gleiche Bezeichnung wie der Parameter "millneigh" haben
```suggestion
  def millneigh: List[(FieldInterface, FieldInterface)]
```